Cargo.toml

```diff
@@ -45,7 +46,10 @@ rust-crypto = { version = "0.2.36", optional = true }
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
+# TODO: publish to crates.io
+sccache-dist = { path = "sccache-dist" }
```

Comment:
I don't know how you'd prefer to deal with this - I can publish and update this PR? Or you can do it next time you release.

Reply:
Is there a specific reason you're adding this code as a separate crate? It's not really usable standalone, right?

Reply:
Yeah, a separate binary within the same crate is what I had envisioned. I get the compile argument (although incremental compilation should help with that nowadays), but managing crates.io releases for two intertwined crates like that is a PITA. (I've dealt with this with another project of mine.)